test/constants.go

```diff
@@ -0,0 +1,11 @@
+package test
+
+const (
+	// LatestStableMattermostVersion is the most recent stable version of
+	// Mattermost.
+	LatestStableMattermostVersion = "5.19.1"
+	// PeviousStableMattermostVersion is the latest dot release of Mattermost
+	// that is one minor version lower than the latest release.
+	// i.e. It's a typical release that would need to be upgraded from.
+	PeviousStableMattermostVersion = "5.18.2"
```

Comment:
You've got a typo in this constant name. Instead of "P**r**evious" you have "Pevious".

Reply:
I think I have finally defeated the everlasting typo. Please re-review when you can.